stacksCollectible/index.ts

```diff
@@ -115,55 +102,67 @@ async function fetchNftData(
   return { collectionData, nftArray };
 }
 
-function organizeNFTsIntoCollection(
+export function organizeNFTsIntoCollection(
   collectionRecord: Record<string, StacksCollectionData>,
-  nftArray: Array<NonFungibleToken>,
+  nftArray: NonFungibleToken[],
   nftCollectionDataArray: Array<NftCollectionData | undefined>,
 ) {
+  const organized: Record<string, StacksCollectionData> = {};
+
   for (let i = 0; i < nftArray.length; i++) {
     const nft = nftArray[i];
     const principal: string[] = nft.asset_identifier.split('::');
     const contractInfo: string[] = principal[0].split('.');
     const contractId = principal[0];
 
-    if (contractInfo[1] === 'bns') continue; //collection already assigned
+    if (contractInfo[1] === 'bns') {
+      // currently stacks only supports 1 bns name per address
+      organised.bns = {
```

Comment:
Sorry, I used English instead of American spelling 😅 

```suggestion
      organized.bns = {
```